connectors-e2e-test/connectors-e2e-test-agentic-ai/src/test/java/io/camunda/connector/e2e/agenticai/aiagent/langchain4j/outboundconnector/BaseL4JAiAgentConnectorTest.java

```diff
@@ -92,11 +94,13 @@ protected ZeebeTest testBasicExecutionWithoutFeedbackLoop(
       Resource process,
       Function<ElementTemplate, ElementTemplate> elementTemplateModifier,
       String responseText,
+      Map<String, Object> extraProcessVariables,
```

Comment:
🔧 Nit: would it make sense to move this above the `responseText`, so the parameters are logically ordered by input to start the process (process, modified template, extra variables) and then the expected response?